.jshintignore

```diff
@@ -24,6 +24,7 @@ apps/clock/js/ext/**
 apps/clock/js/text_builder.js
 shared/js/uuid.js
 shared/js/intl/l20n*.js
+shared/js/media/jpeg-exif.js
```

Comment:
How hard would it be to make this file pass jshint? Maybe file a followup bug to do that?